src/Url/Params/Params.php

```diff
@@ -5,7 +5,7 @@
 
 namespace Zicht\Bundle\UrlBundle\Url\Params;
 
-use Zicht\Bundle\FrameworkExtraBundle\Util\SortedSetMap;
+use Doctrine\Common\Collections\ArrayCollection;
```

Comment:
Deze hadden we er al uitgegooid, maar bleek hier nog te worden gebruikt :see_no_evil: Ik heb de methodes uit `SortedSetMap` overgenomen en van binnen aangepast.